frontend/src/modules/controllers/TestTool.tsx

```diff
@@ -1,3 +1,8 @@
+/**
+ * @summary Re-usable UI component to perform Test calibration
```

Comment:
What does "test calibration" mean? How do tests (I assume this means unit tests) need to be calibrated? When might we need to modify this file?

Reply:
No, this is system Test Calibration which is displayed in settings `Test & Calibration` Tab. TestTool is here is the wrapper for each Test tool displayed in `Test & Calibration`. Right now it has no associated functionality with it.